### ISSUE TITLE
Search supports some greek letters(η, γ, ε) and not others (α, β, δ)

### ISSUE THREAD

**Issue by BradleyPelton** [Stellarium/stellarium-web-engine#48]
### Summary
TLDR: watch https://gyazo.com/650c2c2c638e21ba06358defa97c32fc
SUMMARY: The search bar will recognize and search celestial bodies with some greek letters, but notably doesnt display results
for others (such as alpha and beta)
NOTES: Purely a stylistic choice here, but "NO RESULTS FOUND" should be displayed somewhere when a search returns no results(in my opinion)

### Expected Behavior
Support the searching of all named celestial bodies by their names(which may or not include greek letters)

### Actual Behavior
Only some letters are recognized and not others

### Steps to reproduce
1.) Attempt to search for "η Car Nebula"
2.) You will find it
3.) Attempt to search for Sirius (α Canis Majoris) using "α Canis"
4.) The search will display no results

### Gif of execution
https://gyazo.com/650c2c2c638e21ba06358defa97c32fc

**Comment by gzotti** [comment]
It may be helpful to generally allow searching for Latin transcriptions of greek letters, plus (non-case-sensitive) IAU abbreviation, like "alpha cma", "gamma cas", "epsilon lyr" etc. Few non-Greeks have Greek letters on their keyboards. It may even be possible to allow 3-letter shortnames like "eps", "omi" etc. (2-letters are not enough given variable star names like "EP Lyr".)
